feature/album/info/src/main/java/ru/stersh/youamp/feature/album/ui/AlbumInfoScreen.kt

```diff
@@ -267,13 +267,19 @@ private fun AlbumSongItem(
     onClick: () -> Unit
 ) {
     ListItem(
+        leadingContent = {
+            Text(text = song.track.toString())
```

Comment:
I think, you should check null and after that draw Text.
`
song.track?.let {
   Text(text = it.toString())
}
`